maps/map04.txt

```diff
@@ -1,3 +1,10 @@
-#######
-.  o  .
-#######
+############
+#o  .     g#
+##### ######
+#..........#
+#..........#
+#..........#
+#..........#
+#..........#
+#..........#
+############
```

Comment:
This is the same as the previous but a bit more interesting. :) Currently it's really hard to go through the small holes. You have to be very precise on timing with that. Maybe we have to tweak the controls a bit...